scripts/build-controller.sh

```diff
@@ -16,7 +16,7 @@ source "$SCRIPTS_DIR/lib/k8s.sh"
 : "${ACK_GENERATE_BIN_PATH:=$BIN_DIR/ack-generate}"
 : "${ACK_GENERATE_API_VERSION:="v1alpha1"}"
 : "${ACK_GENERATE_CONFIG_PATH:=""}"
-: "${K8S_RBAC_ROLE_NAME:="ack-controller-role"}"
+: "${K8S_RBAC_ROLE_NAME:="ack-controller"}"
```

Comment:
I removed the "-role" suffix because it was redundant and I want the three roles that we end up producing for installation instructions to be:

* `ack-controller`
* `ack-user`
* `ack-reader`